build/Localization/Settings/LocConfig.xml

```diff
@@ -5,48 +5,48 @@
     tdbuildteamid="104">
     <File
       location="ColorPicker"
-      path="%LocRoot%\..\..\dev\ColorPicker\Strings\en-us\Resources.resw"/>
+      path="dev\ColorPicker\Strings\en-us\Resources.resw"/>
```

Comment:
Do we need this file? Can we just search the repo for en-us\Resources.resw files? That way we don't need to worry about remembering to update this file when a new control gets added.